packages/appinio_swiper/lib/enums.dart

```diff
@@ -1,53 +1,129 @@
-enum AppinioSwiperState {
```

Comment:
I moved to a sealed class system which allows me to express the activity and store state about it.